.github/workflows/node.js.yml

```
@@ -99,7 +99,7 @@ jobs:
         shell: bash
         run: |
           mkdir -p dist/${{ env.ESM_NAME }}/${{ env.TIMESTAMP }}_${{ github.sha }}
-          mv packages/${{ env.DIR_NAME }}/dist/*.* dist/${{ env.ESM_NAME }}/${{ env.TIMESTAMP }}_${{ github.sha }}/
+          mv dist/*.* dist/${{ env.ESM_NAME }}/${{ env.TIMESTAMP }}_${{ github.sha }}/
```

So my suggestion was to push this into a different directory (name doesn't matter) and then publish that one. Otherwise, we'll effectively be publishing two copies of this ESM

```suggestion
          mv dist/*.* publish/${{ env.ESM_NAME }}/${{ env.TIMESTAMP }}_${{ github.sha }}/
```

---

ok... sounds good